docs-zh/blog/2022.md

```diff
@@ -0,0 +1,61 @@
+---
+author: 'Ce Gao'
+avatar: 'https://www.github.com/gaocegege.png'
+github: 'https://github.com/gaocegege'
+twitter: 'https://twitter.com/gaocegege'
+introduction: 'Ce Gao 是 envd 的维护者之一。与此之外，他还是机器学习基础设施开源项目 Kubeflow 的 Co-chair。他主要关注机器学习的模型训练、自动机器学习等领域。'
+---
+
+# TensorChord: 2022 in Review
+
+尽管 TensorChord 还很年轻，但自去年六月底推出了我们的第一个开源项目 [envd](https://github.com/tensorchord/envd) 以来，短短的六个月内取得了令人惊喜的结果。为了庆祝我们迄今为止取得的进展，并展望 2023 年的前景，我们想分享今年的工作，并简单讨论我们看到的下一步方向。
+
+## 开源社区
+
+2022 年 6 月 18 日，我们开源了 envd 并发布了第一个正式版本 0.1.0。到目前为止，我们收获了来自社区的大量反馈，我们也在不断地改进我们的产品，保持着以一周为单位的迭代周期与版本发布节奏。在开源的一周后，我们合并了第 100 个 PR；在 10 月 21 日，我们获得了第 1024 个 GitHub star。我们的社区正在不断壮大，我们也在不断地学习如何与社区进行有效的沟通。
```

Comment:
`envd`

Reply:
Actually, the Chinese content is also generated by jasper